pkg/sdk/tags_test.go

```diff
@@ -0,0 +1,286 @@
+package sdk
+
+import (
```

Comment:
please add also the validation tests to this file

Reply:
fixed